particles/Processing/js/ImageToTensor.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * @license
+ * Copyright (c) 2019 Google Inc. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * Code distributed by Google as part of this project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+defineParticle(({DomParticle, resolver, log}) => {
+
+  importScripts(resolver(`$here/tf.js`));
+
+  return class extends self.TfMixin(DomParticle) {
```

Comment:
Why do we need to mix in Tf?

Reply:
Does this construct `this.tf`, in some sense?

Reply:
Yes. I didn't like the boilerplate I needed to create the tf instance, so I decided to make it a mixin and take advantage of life-cycle methods (e.g. `constructor`). We can salt to taste.